src/components/organisms/EarthEditor/core/CanvasArea/hooks.ts

```diff
@@ -1,9 +1,11 @@
 import { useMemo, useEffect, useCallback } from "react";
 
+import { ClusterProperty } from "@reearth/components/molecules/Visualizer";
```

Comment:
Can you copy ClusterProperty type here? We already know that Visualizer will be removed in the future, so let's clear up the dependencies beforehand.

Reply:
I fixed in this commit
[fix: import](https://github.com/reearth/reearth-web/pull/430/commits/cc4d7fc81fd4c2cac618c9caf5f51df7f42c9c0f)